mappings/net/minecraft/enchantment/EnchantmentHelper.mapping

```diff
@@ -18,15 +31,28 @@ CLASS net/minecraft/class_1890 net/minecraft/enchantment/EnchantmentHelper
 	METHOD method_8202 getRiptide (Lnet/minecraft/class_1799;)I
 		ARG 0 stack
 	METHOD method_8203 getEquipmentLevel (Lnet/minecraft/class_1887;Lnet/minecraft/class_1309;)I
-		ARG 0 ench
+		COMMENT Returns the highest level of the passed enchantment in the enchantment's
+		COMMENT applicable equipment slots' item stacks.
+		ARG 0 enchantment
+			COMMENT the enchantment
+		ARG 1 entity
+			COMMENT the entity whose equipment slots are checked
+	METHOD method_8204 chooseEquipmentWith (Lnet/minecraft/class_1887;Lnet/minecraft/class_1309;)Ljava/util/Map$Entry;
+		COMMENT Returns a pair of an equipment slot and the item stack in the supplied
+		COMMENT entity's slot, indicating the item stack has the enchantment supplied.
+		COMMENT
+		COMMENT <p>If multiple equipment slots' item stacks are valid, a random pair is
+		COMMENT returned.
+		ARG 0 enchantment
+			COMMENT the enchantment the equipped item stack must have
 		ARG 1 entity
-	METHOD method_8204 getRandomEnchantedEquipment (Lnet/minecraft/class_1887;Lnet/minecraft/class_1309;)Ljava/util/Map$Entry;
+			COMMENT the entity to choose equipments from
 	METHOD method_8205 getKnockback (Lnet/minecraft/class_1309;)I
 		ARG 0 entity
 	METHOD method_8206 getLoyalty (Lnet/minecraft/class_1799;)I
 		ARG 0 stack
-	METHOD method_8209 accept (Lnet/minecraft/class_1890$class_1891;Ljava/lang/Iterable;)V
-		ARG 0 enchantmentHandler
+	METHOD method_8209 forEach (Lnet/minecraft/class_1890$class_1891;Ljava/lang/Iterable;)V
```

Comment:
```suggestion
	METHOD method_8209 forEachEnchantment (Lnet/minecraft/class_1890$class_1891;Ljava/lang/Iterable;)V
```
This method is basically applying an action to every enchantment of every stack in the list.

Reply:
private helper method, also note the first parameter is explicit `EnchantmentHelper.Consumer`, so like for each enchantment consumer in iterable

Reply:
Done.